src/components/ViewModal.module.css

```diff
@@ -0,0 +1,118 @@
+.parent {
+  background: gray;
+  height: 100vh;
+  width: 100%;
+  position: relative;
+}
+
+.active {
+  border: #CBCBCB solid 1px;
+  border-radius: 25px;
+  width: 50%;
+  height: 62vh;
+  position: absolute;
+  top: 20%;
+  left: 25%;
+}
+
+.bodyLeft {
```

Comment:
nit: css classnames should be separated with hyphens so like "body-left and body-right"

Reply:
when i replaced with hyphens and tried to label components with the styles.body-right it was erroring and looking for a 'right' css. i replaced with underscores instead since i couldnt figure out what was going wrong